packages/shared/src/components/modals/PushNotificationModal.tsx

```diff
@@ -1,13 +1,18 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { cloudinary } from '../../lib/image';
 import { Button, ButtonVariant } from '../buttons/Button';
 import { Justify } from '../utilities';
 import { Modal, ModalProps } from './common/Modal';
 import { NotificationPromptSource } from '../../lib/analytics';
 import { usePushNotificationMutation } from '../../hooks/notifications';
+import { useViewSize, ViewSize } from '../../hooks';
+import { Drawer } from '../drawers';
+import ConditionalWrapper from '../ConditionalWrapper';
 
 function PushNotificationModal(modalProps: ModalProps): ReactElement {
   const { onRequestClose } = modalProps;
+  const isMobile = useViewSize(ViewSize.MobileL);
```

Comment:
Same as above, do we need to use the hook to display the drawer instead of CSS media query directly?